particles/Loadmesh.cpp

```diff
@@ -97,9 +97,9 @@ struct meshdata attrib_to_data(tinyobj::ObjReader reader, std::string inputfile,
     return d;
 }
 
-struct meshdata getmesh()
+struct meshdata getmesh(std::string meshName)
 {
-    std::string inputfile = "ponte.obj";
+    std::string inputfile = meshName+".obj";
```

Comment:
Shouldn't this be directly an absolute path ? e.g. just meshName. Because for instance on linux there may be .OBJ files and since the fs is case-sensitive this would fail...